lib/eclipse/tests/ScheduleTests.cpp

```diff
@@ -1171,7 +1171,7 @@ BOOST_AUTO_TEST_CASE(changeBhpLimitInHistoryModeWithWeltarg) {
             " 18  OKT 2008 / \n"
             "/\n"
             "WCONHIST\n"
-            "   'I' 'OPEN' 'RESV' 6*  /\n/\n"
+            "   'P' 'OPEN' 'RESV' 6*  /\n/\n"
```

Comment:
It looks like making `I` to be producer here is on purpose, while the behavoir is difficult here. Some discussion is needed here.  And for this example, an extra group of `DATES` is needed at the end, otherwise schedule is rather incomplete?

Reply:
More things need to be done handle this injector/producer switch, will come back later to address this.

Reply:
Closer looking shows this was intentional in the test, so it was recovered by the later commit.